### PR TITLE
Added new center positions

### PR DESCRIPTION
**Pull request by PatrykSzwer**
For #18

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -153,6 +153,25 @@ <h3><code>position="bottom left, bottom beforeright"</code> (fallback positions
   <div slot="handle">Expand ▶</div>
   <div slot="expandable">Expanded content</div>
 </juicy-popover>
+    
+    
+<h3><code>position="beforetop center"</code></h3>
```
Comment (warpech) [inline index.html]: Works great, but can you please move the demo of `beforetop center` after the demo of `beforetop left`?

File: index.html [hunk 2]
```diff
@@ -153,6 +153,25 @@ <h3><code>position="bottom left, bottom beforeright"</code> (fallback positions
...
+<juicy-popover id="rightexpand" position="beforetop center">
+    <div slot="handle">Expand ▶</div>
+    <div slot="expandable">Expanded content</div>
+</juicy-popover>
+ 
+<h3><code>position="middle center"</code></h3>
```
Comment (warpech) [inline index.html]: Works great, but can you please move the demo of `middle center` after the demo of `middle left`?

File: index.html [hunk 3]
```diff
@@ -153,6 +153,25 @@ <h3><code>position="bottom left, bottom beforeright"</code> (fallback positions
...
+<juicy-popover id="rightexpand" position="middle center">
+    <div slot="handle">Expand ▶</div>
+    <div slot="expandable">Expanded content</div>
+</juicy-popover>
+ 
+<h3><code>position="bottom center"</code></h3>
```
Comment (warpech) [inline index.html]: Works great, but can you please move the demo of `bottom center` after the demo of `bottom left`?

**Comment by alshakero** [comment]
Can you add tests for these positions two please?

File: index.html (hunk 2 above)
Comment (PatrykSzwer) [reply index.html]: There is no middle left demo.

**Comment by PatrykSzwer** [comment]
Done

**Pull request merged**